quic/s2n-quic-rustls/Cargo.toml

```diff
@@ -14,7 +14,7 @@ bytes = { version = "1", default-features = false }
 rustls = { version = "0.20", features = ["quic"] }
 rustls-pemfile = "0.3"
 s2n-codec = { version = "=0.1.0", path = "../../common/s2n-codec", default-features = false }
-s2n-quic-core = { version = "=0.1.3", path = "../s2n-quic-core", default-features = false }
+s2n-quic-core = { version = "=0.2.0", path = "../s2n-quic-core", default-features = false }
```

Comment:
since we are bumping a dependency, the behavior of this crate changes so it should also get bumped.